src/main/MainTabsScreen.js

```diff
@@ -89,31 +89,48 @@ function MainTabsScreen(props: Props) {
           component={HomeScreen}
           options={{
             tabBarLabel: 'Home',
-            tabBarIcon: ({ color }) => <IconInbox size={24} color={color} />,
+            tabBarIcon: properties => (
+              <View style={styles.center}>
+                <IconInbox size={24} color={properties.color} />
+                {properties.focused && <Text style={styles.bottomNav}>Home</Text>}
+              </View>
+            ),
           }}
         />
         <Tab.Screen
           name="stream-tabs"
           component={StreamTabsScreen}
           options={{
             tabBarLabel: 'Streams',
-            tabBarIcon: ({ color }) => <IconStream size={24} color={color} />,
+            tabBarIcon: properties => (
+              <View style={styles.center}>
+                <IconStream size={24} color={properties.color} />
+                {properties.focused && <Text style={styles.bottomNav}>Streams</Text>}
+              </View>
+            ),
           }}
         />
         <Tab.Screen
           name="pm-conversations"
           component={PmConversationsScreen}
           options={{
             tabBarLabel: 'Conversations',
-            tabBarIcon: ({ color }) => <IconUnreadConversations color={color} />,
+            tabBarIcon: properties => (
+              <IconUnreadConversations color={properties.color} focused={properties.focused} />
```

Comment:
Why no label here? Seems like that'd be awfully inconsistent in the UI.

Reply:
It has a label Greg, the problem is that the component `IconUnreadConversations` isn't actually an Icon that accepts text but a view component (that has an icon inside it), so I added the label in IconUnreadConversations.js instead of placing it here!

Reply:
Ahh, I see. But why wouldn't it work to do the same thing for this one as for the others?

As is, it looks like it'd cause the `CountOverlay` to enclose the label as well as the icon. I'd expect that won't look right.

Reply:
> But why wouldn't it work to do the same thing for this one as for the others?

Sorry, Greg! It does work that way. Seems like I've made some mistake previously...
Now it works as expected!